core-web/libs/portlets/dot-ema/src/lib/feature/dot-ema.component.html

```diff
@@ -46,4 +46,11 @@ <h4>{{ vm.pageTitle }}</h4>
             height="100%"></iframe>
         <dot-spinner *ngIf="vm.dialogIframeLoading" data-testId="spinner"></dot-spinner>
     </p-dialog>
+
+    <p-confirmDialog
```

Comment:
Add tests to assert this is in the DOM.

Reply:
Might want to do it for the other dialog as well.

Reply:
Done!